src/rest_settings.ts

```diff
@@ -225,6 +225,170 @@ const paths = {
    * @method get
    */
   queryTokenHistory: '/tokens/history/query',
+
+  // Notifiaction
+
+  /*
+  * Waits for the CronjobCreated notification and returns the result
+  *@tag Notification
+  *@method get
+  */
+  onCronjobCreated: '/notifications/long_polling/onCronjobCreated',
```

Comment:
Bitte entweder snake case, oder camel case, aber nicht gemischt.

Entweder: 

```suggestion
  onCronjobCreated: '/notifications/long_polling/on_cronjob_created',
```

oder:

```suggestion
  onCronjobCreated: '/notifications/longPolling/onCronjobCreated',
```